### DISCUSSION TITLE
如何修改表注释

### DISCUSSION THREAD

**Discussion by lafang2009** [apache/doris#6759] [A - General / Q&A]
网上搜索发现     Doris Weekly：支持通过 alter table 命令修改列注释和表注释。

没有找到具体语句，用mysql的语句（alter table detail comment "测试";）也报错。具体修改表注释的语句能发下吗？

**Comment by pushiwuhua7** [comment]
http://doris.apache.org/master/zh-CN/sql-reference/sql-statements/Data%20Definition/ALTER%20TABLE.html#description
10. 修改表注释
    语法:
        MODIFY COMMENT "new table comment"
11. 修改列注释
    语法:
        MODIFY COLUMN col1 COMMENT "new column comment"
